src/main/java/exception/MarkException.java

```diff
@@ -0,0 +1,14 @@
+package exception;
+
+/**
+ * This exception is thrown when the user is trying to mark a task that has already been marked.
+ */
+public class MarkException extends DukeException {
```

Comment:
For better clarity, maybe prefix all Exception extending DukeException with "Duke". i.e DukeMarkException.